app/src/main/res/xml/remote_theme_prefs.xml

```diff
@@ -2,7 +2,85 @@
 <PreferenceScreen
     xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto">
-    <PreferenceCategory app:title="TODO: MOB-2135">
+
+    <Preference
+        android:title="N.B: Needs restart"
+        android:selectable="false"
+        android:enabled="true"
+        android:summary="Any changes made to remote theme require app restart"/>
```

Comment:
I thought that the remote theme could be applied "on fly", i.e. on screen opening.  Does it really require app restart?

Reply:
You can pass the remote theme also on `Intent` that launches ChatActivity or CallActivity. But that would not work for Call Visualizer engagements so it was simpler for me to add it to SDK `init` method in the Test App so that it would work for all screens and views.